Pocket/pocket-ios
# feat(spotlight): adding in a deeplinks for spotlight

## Summary
* Enable deep linking from core spotlight search results to things in your list. Indexing is done by https://github.com/Pocket/pocket-ios/blob/6d0d6dce14a584171d794c4c44fd5f286a82c82d/PocketKit/Sources/Sync/CoreDataSpotlightDelegate.swift#L10 with CoreData automatically.

## References 
* https://mozilla-hub.atlassian.net/browse/POCKET-9379

## Implementation Details
* I went short/sweet and a bit hacky. Coredata continues the activity with a CoreData URI, so i turn that into an object, grab the url, and then let it run through our router to take a user to the item. I am open to better suggestions here, but this seemed like the easiest path for now.


## Test Steps
* Log into your Pocket account, as items load in, CoreData will auto spotlight index them in out SpotlightCoreDataDelegate.
* Search in spotlight for something in your list.
* Tap it, and you should be taken to the url within your list.

## PR Checklist:

## Screenshots
